custom_components/birthdays/__init__.py

```diff
@@ -13,15 +13,20 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-CONF_NAME = 'name'
-CONF_DATE_OF_BIRTH = 'date_of_birth'
-CONF_ICON = 'icon'
+CONF_UNIQUE_ID            = 'unique_id'
+CONF_NAME                 = 'name'
+CONF_DATE_OF_BIRTH        = 'date_of_birth'
+CONF_ICON                 = 'icon'
+CONF_ATTRIBUTES           = 'attributes'
+CONF_AGE_AT_NEXT_BIRTHDAY = 'age_at_next_birthday'
 DOMAIN = 'birthdays'
 
 BIRTHDAY_CONFIG_SCHEMA = vol.Schema({
-    vol.Required(CONF_NAME): cv.string,
-    vol.Required(CONF_DATE_OF_BIRTH): cv.date,
-    vol.Optional(CONF_ICON, default='mdi:cake'): cv.string,
+    vol.Optional(CONF_UNIQUE_ID, default  = None): cv.string,
+    vol.Required(CONF_NAME)               : cv.string,
+    vol.Required(CONF_DATE_OF_BIRTH)      : cv.date,
+    vol.Optional(CONF_ICON, default       = 'mdi : cake'): cv.string,
```

Comment:
Does 'mdi : cake' work with the spaces in between?

Reply:
Probably not! I guess the auto-align uses `:` as a symbol to put spaces around